### PR TITLE
create ttp

### PR DESCRIPTION
**Pull request by anukaal**

### CODE REVIEW

File: discovery/9fe853c0-2cd7-49dc-88b4-ccbc25029dd7.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+id: 9fe853c0-2cd7-49dc-88b4-ccbc25029dd7
+metadata:
+    version: 1
+    authors:
+        - Anurag (anukaal)
+    tags: []
+name: View the password data
+description: |
+  Linux distributions use the /etc/shadow file to store encrypted password data and can show the name of user .
```
Comment (khyberspache) [inline discovery/9fe853c0-2cd7-49dc-88b4-ccbc25029dd7.yml:9]: > Linux distributions use the /etc/shadow file to store encrypted password data. You will also see names of all users on the system.

File: discovery/9fe853c0-2cd7-49dc-88b4-ccbc25029dd7.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+id: 9fe853c0-2cd7-49dc-88b4-ccbc25029dd7
+metadata:
+    version: 1
+    authors:
+        - Anurag (anukaal)
+    tags: []
+name: View the password data
```
Comment (khyberspache) [inline discovery/9fe853c0-2cd7-49dc-88b4-ccbc25029dd7.yml:7]: View linux shadow file

File: discovery/16fd2706-8baf-433b-82eb-8c7fada847da.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+id: 16fd2706-8baf-433b-82eb-8c7fada847da
+metadata:
+    version: 1
+    authors:
+        - Anurag (anukaal)
+    tags: []
+name: View the bashrc file
```
Comment (khyberspache) [inline discovery/16fd2706-8baf-433b-82eb-8c7fada847da.yml:7]: View system-wide bashrc configuration

File: discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+id: 00010203-0405-0607-0809-0a0b0c0d0e0f
+metadata:
+    version: 1
+    authors:
+        - Anurag (anukaal)
+    tags: []
+name: View the hidden file
```
Comment (khyberspache) [inline discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml:7]: View hidden files in working directory

File: discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+description: |
+   ls -a in linux will show you the hidden files in your system.
```
Comment (khyberspache) [inline discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml:9]: > Use the ls tool to view all files (including hidden files) in the working directory

**Review by khyberspache — CHANGES_REQUESTED** [review]

**Comment by anukaal** [comment]
Done all the changes

File: discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml (hunk 4 above)
Comment (khyberspache) [reply discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml:7]: Once this is changed, I can merge.

**Review by khyberspache — CHANGES_REQUESTED** [review]

File: discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml (hunk 4 above)
Comment (anukaal) [reply discovery/00010203-0405-0607-0809-0a0b0c0d0e0f.yml]: Done

**Review by khyberspache — APPROVED** [review]

**Pull request merged**